### DISCUSSION TITLE
ED2E8A2D-E585-42C7-A697-D8F2F2C4F471

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#19] [General]
# ED2E8A2D-E585-42C7-A697-D8F2F2C4F471

HD 40307 g is the possible but disputed outermost planet of HD 40307. It orbits within the habitable zone and is listed as a potentially habitable planet, but given its large mass it's unlikely to have a rocky composition.

https://celestia.mobi/addon?item=ED2E8A2D-E585-42C7-A697-D8F2F2C4F471

**Comment by exocritic5457** [comment]
Not to be that one guy but either this (candidate) planet's eccentricity was forgotten, or all of those orbital diagrams of HD 40307 g were wrong.
